### ISSUE TITLE
Clarification on Logits Output and Class Mapping in Non-Verbal Vocalization Model

### ISSUE THREAD

**Issue by Choddeok** [koudounasalkis/voc2vec#1]
Dear authors,

First of all, thank you for sharing such an excellent paper along with the code and pretrained models.
Thanks to the well-structured implementation and clear documentation, I was able to run the code smoothly.

I have a few questions I hope you could kindly clarify.

When I run inference using logits = model(**inputs).logits, the output is as follows:

```
output: SequenceClassifierOutput(loss=None, logits=tensor([[-0.0214,  0.0213]], grad_fn=<AddmmBackward0>), hidden_states=None, attentions=None)
logits: tensor([[-0.0067,  0.0160]], grad_fn=<AddmmBackward0>)
```

Could you please explain in more detail what these logits represent in the context of the non-verbal vocalization classification?
It would be very helpful if you could provide more information about the specific classes used for fine-tuning and how the logits correspond to each class.

Additionally, have you experimented with feeding verbal vocalizations (i.e., speech or spoken utterances) into the model? If so, I’d greatly appreciate it if you could share any insights or results from those experiments.

Thank you again for your valuable work!

**Comment by MorenoLaQuatra** [comment]
Thank you for your interest in our work!

The model you're using is *pre-trained only* and has not been fine-tuned for any specific classification task. When you instantiate it with a `SequenceClassificationHead`, Hugging Face automatically initializes it with a randomly initialized linear classification head with **2** classes by default. If you want to just extract representations you can just use the `AutoModel` class.

TL;DR: The logits you're seeing have no real meaning since the classification head is randomly initialized.

If you want to extract meaningful representations from the pre-trained model, you should use:
- `last_hidden_state`: shape `(batch_size, time_frames, model_dim)` 
- `hidden_states`: tuple of outputs from each transformer layer, each with shape `(batch_size, time_frames, model_dim)`

These representations can then be used for downstream tasks or as features for your own classification head. Regarding verbal vocalizations: this model was specifically designed for non-verbal vocalizations, so we haven't systematically evaluated it on speech/spoken utterances.

**Comment by Choddeok** [comment]
Thank you so much for the detailed explanation — that really helped clarify things for me. I truly appreciate your kind response and the time you took to explain it.

Also, once again, thank you for sharing such an insightful piece of research and making your work publicly available. It’s been very inspiring to study and learn from it!
